### ISSUE TITLE
about the dataset

### ISSUE THREAD

**Issue by Polycoriander** [prov-gigapath/prov-gigapath#22]
Hello, Thanks for your nice work. I have some questions about the datasets.
1.I want to reproduce the results from Table 2 in the paper. How can I obtain the datasets related to these results?
2.You've indicated that a download location for the TCGA-LUAD dataset was shared, which is separated into open and controlled sections. Did you utilize only the open part, or did your analysis also incorporate the controlled portion of the dataset?And how to split it when using the 10-fold cross-validation?
3.The paper does not mention the PCam Dataset and PANDA Dataset. What are they used for, and are they merely examples of fine-tuning?
Thank you very much!

**Comment by HanwenXuTHU** [comment]
Hi, thanks for the kind words. 

1. Glad to hear that you're interested in reproducing Table 2. These tasks, except the TCGA-LUAD mutation prediction, are established on the proprietary Prov-Path dataset, which cannot be made publicly available. But we are indeed discussing and working to see if we can release more subset for people to test. Please stay tuned.
2. In our TCGA-LUAD tasks, please find the slide ids we used [here](https://github.com/prov-gigapath/prov-gigapath/blob/main/dataset_csv/mutation/LUAD-5-gene_TCGA.csv). When doing the cross-validation, each time we re-split the train/val/test set based on patient-level stratification.
3. Yes, they are examples to show how to do fine-tuning and linear probing with GigaPath. They are also a starting point for people who are interested in exploring GigaPath on public benchmarks.

Please let me know if there are other issues I can help.

**Comment by Polycoriander** [comment]
Thank you for your detailed response. I am wondering if there are any results available using GigaPath model on the PANDA Dataset and PCam Dataset.

**Comment by HanwenXuTHU** [comment]
In my test, the results are: (1) Weighted F1 of 0.94 on PCam (2) Quadratic weighted kappa of 0.96 on PANDA. These can be some reference numbers to reproduce.

**Comment by Polycoriander** [comment]
Thank you very much for your answer, I have learned a lot.
